doc/source/mijnafvalwijzer_nl.md

```diff
@@ -0,0 +1,40 @@
+# Afval Wijzer
+
+Afval Wijzer is no longer supported by the generic [ICS](/doc/source/ics.md) source. For all available configuration options, please refer to the source description.
```

Comment:
> For all available configuration options, please refer to the source description.

Does not make any sense anymore as THIS is the source description

Reply:
Changed md file with more info